tests/check/check.go

```diff
@@ -197,6 +197,9 @@ func runTest(t *testing.T, test individualTest, params testParams, contextTupleT
 				}
 			}
 
+			if len(stage.CheckAssertions) == 0 {
+				t.Skipf("no check assertions defined")
```

Comment:
Instead of skipping I wonder if we should emit a warning or something or just panic. I'll miss this in the verbose output of our tests.

I think it'd be smart to `require.GreaterEqual(t, stage.CheckAssertions, 1)`.

Reply:
> if we should emit a warning 

that's what this is doing, but it only prints when ran with `-v`. I don't know how to log in every case. There is no `t.Warn`..

> just panic

i thought this. But too many tests only have Check or ListObjects assertions.

```
[30/11/23 1:49:23] ~/GitHub/openfga/tests/check (fix-assertions) $ go test -race . -v -run=TestCheckMemory | grep "no check assertions defined" -B 1
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_error_if_unknown_relation_in_request
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_error_if_unknown_type_in_request
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/error_if_contextual_tuples_do_not_follow_type_restrictions
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_ignores_irrelevant_tuples_because_different_user
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_ignores_duplicate_contextual_tuples
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/ignores_irrelevant_contextual_tuples_because_different_type
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_considers_input_contextual_tuples
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/ignores_irrelevant_contextual_tuples_because_different_user
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/validation_type_not_in_model
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/validation_type_not_in_model_ctxTuples
    check.go:201: no check assertions defined
--
=== CONT  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_expands_wildcard_tuple
    check.go:201: no check assertions defined
--
=== CONT  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_does_not_return_duplicates
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_with_subcheck_encounters_cycle
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_with_subcheck_resolution_depth_exceeded_ctxTuples
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_with_subcheck_encounters_cycle_ctxTuples
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/follows_correct_graph_edges_ctxTuples
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/follows_correct_graph_edges
    check.go:201: no check assertions defined
--
=== NAME  TestCheckMemory/RunAllTests/Check/Schema1_1/list_objects_with_subcheck_resolution_depth_exceeded
    check.go:201: no check assertions defined
[30/11/23 1:50:13] ~/GitHub/openfga/tests/listobjects (fix-assertions) $ go test -race . -v -run=TestListObjectsMemory | grep "no list objects assertions defined" -B 1
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/direct_relations_with_condition_ctxTuples
    listobjects.go:168: no list objects assertions defined
--
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/relation_through_ttu_with_condition_ctxTuples
    listobjects.go:168: no list objects assertions defined
--
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/relation_through_ttu_with_condition
    listobjects.go:168: no list objects assertions defined
--
=== CONT  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/validation_empty_object_ctxTuples
    listobjects.go:168: no list objects assertions defined
--
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/validation_nil_tuple_ctxTuples
    listobjects.go:168: no list objects assertions defined
=== CONT  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/validation_empty_object
    listobjects.go:168: no list objects assertions defined
--
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/direct_relations_with_condition
    listobjects.go:168: no list objects assertions defined
--
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/list_objects_error_if_unknown_relation_in_request
    listobjects.go:168: no list objects assertions defined
--
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/validation_empty_tuple_ctxTuples
    listobjects.go:168: no list objects assertions defined
--
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/validation_empty_tuple
    listobjects.go:168: no list objects assertions defined
--
=== NAME  TestListObjectsMemory/RunAll/ListObjects/Schema1_1/validation_nil_tuple
    listobjects.go:168: no list objects assertions defined

```